waku/factory/external_config.nim

```diff
@@ -694,20 +703,36 @@ proc parseCmdArg*[T](_: type seq[T], s: string): seq[T] {.raises: [ValueError].}
 proc completeCmdArg*(T: type crypto.PrivateKey, val: string): seq[string] =
   return @[]
 
-proc parseCmdArg*(T: type ProtectedTopic, p: string): T =
+# TO DO: Remove when removing protected-topic configuration
```

Comment:
Super nitpick one hehe. We tend to use the compacted version
```suggestion
# TODO: Remove when removing protected-topic configuration
```

Reply:
Oops, true!